### PR TITLE
Jorge/egeitowo00a

### PR DESCRIPTION
**Pull request by jorgefandinno**

### CODE REVIEW

**Comment by jorgefandinno** [comment]
The test fails, but passes in my computer.

Any idea?

File: procs.bib [hunk 1]
```diff
@@ -1579,6 +1579,21 @@ @proceedings{gecco09
...
+T. Son},
+  booktitle = {Logic Programming, Knowledge Representation, and Nonmonotonic Reasoning
+- Essays Dedicated to Michael Gelfond on the Occasion of His 65th
+Birthday},
+  publisher = springer,
+  series = lncs,
+  volume = {6565},
+  year = {2011}
+}
+
```
Comment (tortinator) [inline procs.bib:1596]: Please use `mg65`

File: krr.bib [hunk 2]
```diff
@@ -23219,6 +23219,14 @@ @article{truszczynski11a
   year = {2011}
 }
 
+@inproceedings{truszczynski11b,
+  title = {Revisiting Epistemic Specifications},
+  author = {M. Truszczynski},
+  crossref = {gelfond11},
```
Comment (tortinator) [inline krr.bib:23225]: See below

**Comment by rkaminsk** [comment]
> The test fails, but passes in my computer.

You probably have to update the bibtexparser to version 1.3.

**Comment by jorgefandinno** [comment]
> > The test fails, but passes in my computer.
> 
> You probably have to update the bibtexparser to version 1.3.

Thanks

**Review by tortinator — APPROVED** [review]

**Pull request merged**